### DISCUSSION TITLE
https://theme-hope.vuejs.press/guide/markdown/chart/plantuml.html

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#886] [Announcements]
# https://theme-hope.vuejs.press/guide/markdown/chart/plantuml.html

Let the Markdown files in your VuePress site support plantuml. Settings You can enable this feature via: Syntax You can insert the same content that plantuml supports, for examp...

https://theme-hope.vuejs.press/guide/markdown/chart/plantuml.html

**Comment by leandro-costa** [comment]
Why not use the same syntax as mermaid?


```markdown

```plantuml

@startuml
content
@enduml

 ```

```

**Reply by Mister-Hope** [reply]
Then why don't you use mermaid？

The current feature is a cilent free feature,  and will not introduce any extra size or extra computation at browser side

**Reply by leandro-costa** [reply]
I prefer plantuml.  It has a wide variety of diagrams.
the code syntax (` ```platuml ``` `) helps  in vscode preview

used it like this

```javascript 
import umlPlugin from 'markdown-it-plantuml';
...
export default defineUserConfig({
...
  extendsMarkdown: (md) => {    
    md.use(umlPlugin, {openMarker: '```plantuml', closeMarker: '```'})
  },
...
});
```

**Reply by leandro-costa** [reply]
How could I now configure it to have this same behavior in my project?
